app/Model/Server.php

```diff
@@ -4450,6 +4450,44 @@ public function removeOrphanedCorrelations()
         return $success;
     }
 
+    public function queryAvailableSyncFilteringRules($server)
+    {
+        $HttpSocket = $this->setupHttpSocket($server, null);
+        $uri = $server['Server']['url'] . '/servers/getAvailableSyncFilteringRules';
```

Comment:
We have some options to mitigate the lack of this function on the remote side (if we want that, the alternative is to just not care and display that this feature only works for MISP > v2.4.142):

Using /organisations/index in junction with /tags/index. Though both the query and the response will be heavier this way.